cpu/atmega328p/Kconfig

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config CPU_SERIES_ATMEGA328
+    bool
+    select CPU_FAMILY_ATMEGA
+    select HAS_ATMEGA_PCINT1
+    select HAS_ATMEGA_PCINT2
+
+## CPU Models
+config CPU_MODEL_ATMEGA328P
+    bool
+    select CPU_SERIES_ATMEGA328
+    select HAS_CPU_ATMEGA328P
+
+## Definition of specific features
+config HAS_CPU_ATMEGA328P
+    bool
+    help
+        Indicates that a 'atmega328p' cpu is being used.
+
+## Common CPU symbols
+config CPU_SERIES
```

Comment:
What does CPU_SERIES relate to?

Reply:
There is no definition in `vars.inc.mk`

Reply:
We are trying to have the structure described in https://github.com/RIOT-OS/RIOT/blob/master/cpu/Kconfig

Reply:
Ok for `CPU_SERIES`

Reply:
But those definitions are quite vague to me. Is it on purpose to have them arbitrary like that?
Also `CPU` is not in the graph. I'm again jumping super late into this, but would be better if the descriptions matched `vars.inc.mk`. This is of course for another PR.

Reply:
> But those definitions are quite vague to me. Is it on purpose to have them arbitrary like that?

Do you mean the help strings? We could improve them in `cpu/Kconfig`.

CPU is actually there to mirror the current CPU variable, which is more of a buildsystem variable to find the correct folder.

Reply:
Yes the help string.

Reply:
But this would be another PR, I'll summarize on my final review message.